include/onnxruntime/core/common/inlined_containers_fwd.h

```diff
@@ -65,7 +65,7 @@ struct CalculateInlinedVectorDefaultInlinedElements {
   // 1. There is at least one inlined element.
   // 2. `sizeof(InlinedVector<T>) <= kPreferredInlinedVectorSizeof` unless
   // it contradicts 1.
-  static constexpr size_t kPreferredInlinedVectorSizeof = 128;
```

Comment:
> size_t kPreferredInlinedVectorSizeof = 128; [](http://example.com/codeflow?start=18&length=44)

You do not want to do it in the future w/o running perf tests as it has a global impact.

Reply:
I am actually seeing it is 64 in the main, [https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/common/inlined_containers_fwd.h#L68](https://github.com/microsoft/onnxruntime/blob/main/include/onnxruntime/core/common/inlined_containers_fwd.h#L68)